src/main/webapp/app/app.module.ts

```diff
@@ -4,9 +4,10 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
```

Comment:
I skipped the frontend review remarks in general. unfortunately there are also some things to note